### PR TITLE
created dotPDN.paint.net (manifest 1.4)

### PR DESCRIPTION
**Pull request by licon4812**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

**I have prepared this PR for when packages made with manifest 1.4 are supported**

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82875)

### CODE REVIEW

**Comment by licon4812** [comment]
I have prepared this PR for when manifest 1.4 is supported

File: manifests/d/dotPDN/paintdotnet/4.3.12/dotPDN.paint.net.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+PackageIdentifier: dotPDN.paintdotnet
+PackageVersion: 4.3.12
+PackageLocale: en-US
+Publisher: dotPDN LLC
+PublisherUrl: https://www.getpaint.net/index.html
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: dotPDN
+PackageName: Paint.net
```
Comment (ItzLevvie) [inline manifests/d/dotPDN/paintdotnet/4.3.12/dotPDN.paint.net.locale.en-US.yaml:12]:
```suggestion
PackageName: paint.net
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
I'd also like to note that this doesn't contain all of the installer types that paint.net has. See https://github.com/microsoft/winget-pkgs/compare/master...Trenly:winget-pkgs:zips/paintdotnet-4.3.12?expand=1 for a complete manifest

**Comment by licon4812** [comment]
> I'd also like to note that this doesn't contain all of the installer types that paint.net has. See https://github.com/microsoft/winget-pkgs/compare/master...Trenly:winget-pkgs:zips/paintdotnet-4.3.12?expand=1 for a complete manifest

Since you have a PR already ready to go, would you like me to close this one?

**Pull request closed without merging**